Sources/FirebaseFirestore/CollectionReference.swift

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
This is a set of Swift extensions to the C++ type right?  I think that `CollectionReference+Swift.swift` might be the nicer name.